app/javascript/components/pages/ApartmentShow.js

```diff
@@ -21,6 +21,8 @@ export default class ApartmentShow extends Component {
                 <br/>
                 <span>{apartment.pets}</span>
               </CardText>
+              <Button><a href={`/apartmentindex`} >Back to Listings</a></Button>
+              <Button><a href={`/apartmentedit`} >Edit Listing</a></Button>
```

Comment:
You may need more information on this route to say which apartment you want to edit.